docs/examples/objc/iOS-Carthage/Cartfile

```diff
@@ -1 +1,2 @@
-github "cossacklabs/themis" ~> 0.14.0
\ No newline at end of file
+github "cossacklabs/themis" "0.15.1"
```

Comment:
0.15.1 or 0.15.2?

Reply:
Apple-related wrappers have version 0.15.1
Would you recommend increasing all package versions to 0.15.2 without any changes?